modules/mirror-registry-remote.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-mirroring-installation-images.adoc
+
+[id="mirror-registry-remote_{context}"]
+= Mirroring on a remote host with mirror registry for {product-title}
+
+Remote mirroring with mirror registry for {product-title}
+
+This procedure introduces how to install Red Hat Quay on a remote host using the `mirror-registry` tool. By doing so, users can create a registry to hold a mirror of {product-title} images.
+
+[NOTE]
+====
+Installing Red Hat Quay using the `mirror-registry` CLI tool makes several changes to your machine. After installation, a `/etc/quay-install` directory is created, which has installation files, local storage, and the configuration bundle. Trusted SSH keys are generated in case the deployment target is the local host, and systemd files on the host machine are set up to ensure that container runtimes are persistent. Additionally, an initial user named `init` is created with an auto-generated password. All access credentials are printed at the end of the install routine.
+====
+
+.Procedure
+
+. Download the `mirror-registry.tar.gz` package for the latest version of mirror registry found on the link:https://console.redhat.com/openshift/downloads[*Downloads*] page of {product-title}.
```

Comment:
We need to mention online and offline variants of the tarball here.

Reply:
My understanding is that we’re definitely not building the online installer downstream

Reply:
Update: my understanding after a conversation with Dave is that there is only a single download pack available. I've made changes accordingly, to address a single download package (which is the offline package that pulls the required images if `image-archive.tar` isn't available).